python/ql/src/Security/CWE-022/PathInjection.ql

```diff
@@ -4,7 +4,7 @@
  * @kind problem
  * @problem.severity error
  * @sub-severity high
- * @precision medium
+ * @precision high
```

Comment:
This should have a change note since it changes the behaviour of alerts from these queries.